app/lib/backend_api_logic/service_extension.rb

```diff
@@ -21,6 +21,14 @@ def find_or_create_first_backend_api!
       config.backend_api
     end
 
+    def acts_as_product?
```

Comment:
I would rename this to the same name of the column `act_as_product?` so it's not confusing. Plus, when/if we remove the rolling update, it will be as simple deleting this method.